buildpacks/java/go.mod

```diff
@@ -1,11 +1,50 @@
 module kn-fn/java-function-buildpack
 
-go 1.16
+go 1.19
 
 require (
 	github.com/buildpacks/libcnb v1.26.0
 	github.com/paketo-buildpacks/libpak v1.61.0
-	gopkg.in/yaml.v3 v3.0.1
-	knative.dev/kn-plugin-func v0.19.0
-	knative.dev/pkg v0.0.0-20210902173607-844a6bc45596
+	kn-fn/shared v0.0.0
```

Comment:
🤔

Reply:
I turns out it just needs to be any semver. I don't have a preference as to what value we use, since it's purely internal to this repo.